packages/kbn-server-route-repository/src/register_routes.ts

```diff
@@ -63,6 +63,7 @@ export function registerRoutes({
             request,
             context,
             params: validatedParams,
+            logger,
```

Comment:
not sure if this is correct that this file is owned by UX OBST MANAGEMNT TEAM !!

Reply:
Paul put it there, I'll remove it.